dev/manual_tests/lib/text.dart

```diff
@@ -181,22 +179,22 @@ class _FuzzerState extends State<Fuzzer> with SingleTickerProviderStateMixin {
     if (_random.nextInt(200) == 0)
       return null;
     return TextStyle(
-      color: _fiddleWithColor(style.color),
-      decoration: _fiddleWithDecoration(style.decoration),
-      decorationColor: _fiddleWithColor(style.decorationColor),
-      decorationStyle: _fiddleWithDecorationStyle(style.decorationStyle),
-      fontWeight: _fiddleWithFontWeight(style.fontWeight),
-      fontStyle: _fiddleWithFontStyle(style.fontStyle),
+      color: _fiddleWithColor(style.color!),
```

Comment:
All the `_fiddle` methods should take nullable parameters, they seem to handle null?